### DISCUSSION TITLE
Feature request: Compare time periods

### DISCUSSION THREAD

**Discussion by Fawesum** [umami-software/umami#1819] [Ideas]
It would be really handy to have a time comparison feature where I could compare say, the last week with the week before, or months up against each other. Or just any time period put up against another. Would pretty much bring Umami up to parity with Google Analytics for my usuage!

**Comment by mikecao** [comment]
Can you post a screenshot of what you expect? Not sure I understand completely.

**Comment by Fawesum** [comment]
Sure! Here's the equivalent in Google Analytics where I can choose two date ranges and get a comparison:

![2021-04-09 13_09_42-Window](https://user-images.githubusercontent.com/7153991/114171961-5303b380-9935-11eb-8ef6-db3994a02b6f.jpg)

![2021-04-09 13_12_21-Window](https://user-images.githubusercontent.com/7153991/114171967-5434e080-9935-11eb-93f2-399824556060.jpg)

Basically just getting two data sets, then running a calculation on the newest data set vs the oldest one.
Having a visual representation isn't really that important though.

**Comment by abrambailey** [comment]
This is the main reason we aren't considering umami.

**Comment by wc-matteo** [comment]
plausible's got it, shame umami hasn't
